app/controllers/users/sessions_controller.rb

```diff
@@ -1,11 +1,25 @@
 # frozen_string_literal: true
 
 class Users::SessionsController < Devise::SessionsController
-  skip_before_action :authenticate_user!, only: %i[new destroy]
+  include SessionTimeoutConcern
```

Comment:
I'm wondering if we get much benefit from introducing a concern if it's only used in one place, it might be fine to include the methods directly in to this controller.

Reply:
Yes, fair enough. Done in [58c545c](https://github.com/nhsuk/manage-vaccinations-in-schools/pull/4949/commits/58c545c5669b64a68b99136b29de94329599a47a).